### PR TITLE
staticanalysis/bot: Add more metric around mach steps(#1053)

### PR DESCRIPTION
**Pull request by rv404674**
hey @La0 , i have added metric around all mach steps which i could find, so can you review it. :)

### CODE REVIEW

**Comment by La0** [comment]
Hello @rv404674, thanks for your PR.

The indentation seems wrong (see the log output [from the CI build](https://tools.taskcluster.net/groups/ZReEFCisTOuDyH0upuR4ng/tasks/N2XGjYrWQ3un9ROK2edVQA/runs/0/logs/public%2Flogs%2Flive.log#L2099))

Also, this PR is not about Infer, but to cover [these mach steps](https://github.com/mozilla/release-services/blob/master/src/staticanalysis/bot/static_analysis_bot/workflow.py#L162).
Can you cover each of them with a timer ?

**Comment by rv404674** [comment]
Thanks for the review @La0 , PR is completed now.

**Comment by rv404674** [comment]
hey @jankeromnes , the test are failing.

**Comment by La0** [comment]
Please use a unique name for each timer. The name has to be relevant to the mach command.
For example, the `configure` command would give the timer name `runtime.mach.configure`.

Right now the PR is useless as it will overwrite each metric with the same name.

File: src/staticanalysis/bot/static_analysis_bot/workflow.py [hunk 1]
```diff
@@ -200,7 +204,8 @@ def run(self, revision):
             # Setup python environment
             logger.info('Mach lint setup...')
             cmd = ['gecko-env', './mach', 'lint', '--list']
-            out = run_check(cmd, cwd=settings.repo_dir)
+            with stats.api.timer('runtime.mach.list'):
```
Comment (La0) [inline src/staticanalysis/bot/static_analysis_bot/workflow.py]: Please use `runtime.mach.lint` instead of `runtime.mach.list` here

**Pull request merged**